LICENSE.3rdparty

```diff
@@ -0,0 +1,26 @@
+Portions of this code were took from the Guzzle PHP HTTP client project (https://guzzlephp.org/en/stable/) and are available under the following terms:
```

Comment:
It would be great if this license notice could got to the related files, because currently it's not clear which parts are copied. I'm also not sure if this are "substantial portions of the Software".

Reply:
I think the correct url is https://docs.guzzlephp.org/en/stable/

Reply:
> I'm also not sure if this are "substantial portions of the Software".

I am not, too. As I am not lawyer, I would not try to interpret these conditions and always add the hint to used code ;-) .

> It would be great if this license notice could got to the related files, because currently it's not clear which parts are copied.

Done.